apps/vaporgui/PImportDataButton.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "PLineItem.h"
+
+namespace VAPoR {
+    class ControlExec;
+}
+class MainForm;
+class VHBoxWidget;
+class VLabel;
+class PButton;
+
+class PImportDataButton : public PWidget {
```

Comment:
The import data button probably doesn't need to represent state within the database so why make a it params widget?